app/build.gradle

```diff
@@ -13,8 +13,8 @@ android {
         applicationId 'org.openobservatory.ooniprobe'
         minSdk libs.versions.minSdk.get().toInteger()
         targetSdk libs.versions.targetSdk.get().toInteger()
-        versionName '3.8.5'
-        versionCode 107
+        versionName '3.8.5.1'
```

Comment:
Can you make it 3.8.6? I am not 100% sure we should deviate from three version codes and semver.

Reply:
I am not. this patch is already live for a quick fix so I don't know how that would go.